Gemfile.lock

```diff
@@ -0,0 +1,32 @@
+PATH
+  remote: .
+  specs:
+    slop (4.3.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    byebug (11.1.3)
+    coderay (1.1.3)
+    method_source (1.1.0)
+    minitest (5.0.8)
+    pry (0.14.2)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry-byebug (3.10.1)
+      byebug (~> 11.0)
+      pry (>= 0.13, < 0.15)
+    rake (13.2.1)
+
+PLATFORMS
+  ruby
```

Comment:
i have `arm64-darwin-23` in my file, does it make sense to add or not make a diff either (basically same q as below, coming from my v little Gemfile knowledge 🙃)

Reply:
In theory bundler generates platforms automatically based on the platform support that the dependencies specify, so afaik we should just stick with what this has currently rather than manually adding anything.